### ISSUE TITLE
Support for webhooks?

### ISSUE THREAD

**Issue by bobbytables** [heroku/platform-api#87]
This looks to be pretty out of date, are there plans to add the endpoints like webhooks to this?

**Comment by we4tech** [comment]
👍

**Comment by we4tech** [comment]
Hi @bobbytables I needed this in our company as well. I just sent a PR with updated schema. Please review this PR https://github.com/heroku/platform-api/pull/88

**Comment by Porkechebure** [comment]
> This looks to be pretty out of date, are there plans to add the endpoints like webhooks to this?

NO, you have to use Gun Jack on Tekken he is the strongest for sure
